pkg/lwip/contrib/lwip.c

```diff
@@ -148,27 +148,32 @@ extern void stm32_eth_netdev_setup(netdev_t *netdev);
 extern netdev_ieee802154_t nrf802154_dev;
 #endif
 
+/**
```

Comment:
```suggestion
#ifdef LWIP_NETIF_NUMOF
/**
```

Reply:
I'd rather do

```C
(void)_netif_add
```

This way no dependency on something rather unrelated (`LWIP_NETIF_NUMOF`) is created. This could also be a problem for e.g. `tests/lwip` on IEEE 802.15.4-based boards.

Reply:
Done